tests/qos/qos_sai_base.py

```diff
@@ -1232,7 +1232,7 @@ def resetWatermark(
         duthost = duthosts[rand_one_dut_hostname]
         dut_asic = duthost.asic_instance(enum_frontend_asic_index)
         dut_asic.command("counterpoll watermark enable")
-        dut_asic.command("sleep 20")
+        dut_asic.command("sleep 70")
```

Comment:
@kevinskwang , have you verified that this fixes the issue?

Reply:
Actually, this issue is flaky, it just appears in the regression test at some nights. I didn't reproduce it with the manual run. So the plan here is to merge it and check the results in the nightly test.